net-core/Ical.Net/Ical.Net/General/CalendarObject.cs

```diff
@@ -150,53 +146,26 @@ public virtual Calendar Calendar
 
         public virtual int Column { get; set; }
 
-        public virtual object GetService(Type serviceType)
-        {
-            return _serviceProvider.GetService(serviceType);
-        }
+        public virtual object GetService(Type serviceType) => _serviceProvider.GetService(serviceType);
 
-        public virtual object GetService(string name)
-        {
-            return _serviceProvider.GetService(name);
-        }
+        public virtual object GetService(string name) => _serviceProvider.GetService(name);
 
-        public virtual T GetService<T>()
-        {
-            return _serviceProvider.GetService<T>();
-        }
+        public virtual T GetService<T>() => _serviceProvider.GetService<T>();
 
-        public virtual T GetService<T>(string name)
-        {
-            return _serviceProvider.GetService<T>(name);
-        }
+        public virtual T GetService<T>(string name) => _serviceProvider.GetService<T>(name);
 
-        public virtual void SetService(string name, object obj)
-        {
-            _serviceProvider.SetService(name, obj);
-        }
+        public virtual void SetService(string name, object obj) => _serviceProvider.SetService(name, obj);
 
-        public virtual void SetService(object obj)
-        {
-            _serviceProvider.SetService(obj);
-        }
+        public virtual void SetService(object obj) => _serviceProvider.SetService(obj);
 
-        public virtual void RemoveService(Type type)
-        {
-            _serviceProvider.RemoveService(type);
-        }
+        public virtual void RemoveService(Type type) => _serviceProvider.RemoveService(type);
 
-        public virtual void RemoveService(string name)
-        {
-            _serviceProvider.RemoveService(name);
-        }
+        public virtual void RemoveService(string name) => _serviceProvider.RemoveService(name);
 
         public virtual string Group
         {
             get { return Name; }
-            set
-            {
-                Name = value;
-            }
+            set { Name = value; }
```

Comment:
You didn't quite go far enough with C# 7's expression bodied members:

https://docs.microsoft.com/en-us/dotnet/articles/csharp/whats-new/csharp-7#more-expression-bodied-members